src/processing.ts

```diff
@@ -104,24 +118,10 @@ export const configurableProcessSpace = async (deps: ConfigurableProcessSpaceDep
   };
   emitter.emit(BotEventNames.SPACE_EVENTS_FETCHED, emittedContext);
 
-  if (proposals.length !== 0 || answers.length !== 0) {
-    await Promise.all([
-      ...proposals.map((event) =>
-        notifyFn({
-          type: EventType.PROPOSAL_QUESTION_CREATED,
-          event,
-          space,
-        }),
-      ),
-      ...answers.map((event) =>
-        notifyFn({
-          type: EventType.NEW_ANSWER,
-          event,
-          space,
-        }),
-      ),
-    ]);
+  await processProposalsFn(space, proposals);
+  await processAnswersFn(space, answers);
```

Comment:
These two async calls can't be done concurrently: Answers processing required proposals processing to work correctly.